lib/pm2/run-staging.sh

```
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Navigate to the intended path
+cd /home/ds/Documents/drop-shot-streaming-scripts-ubuntu/ || exit 1
+
+# Load nvm
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./nvm.sh was not specified as input (see shellcheck -x).

(SC1091)